modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/NewCopyWithDefaultVersion.java

```diff
@@ -72,8 +72,11 @@ public NewCopyWithDefaultVersion(TestUserMode userMode) {
 
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
-        super.init();
-        restAPIPublisher = new RestAPIPublisherImpl();
+        super.init(userMode);
+        String userName = gatewayContextWrk.getContextTenant().getTenantAdmin().getUserNameWithoutDomain();
+        String password = gatewayContextWrk.getContextTenant().getTenantAdmin().getPassword();
+        String tenantDomain = gatewayContextWrk.getContextTenant().getDomain();
+        restAPIPublisher = new RestAPIPublisherImpl(userName, password, tenantDomain);
```

Comment:
Do we need this? This is already done in the Super class.

Reply:
fixed